ops/internal/manage/staging_test.go

```diff
@@ -0,0 +1,30 @@
+package manage
+
+import (
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-chain-ops/genesis"
+	"github.com/ethereum-optimism/superchain-registry/ops/internal/config"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/stretchr/testify/require"
+)
+
+func TestCopyDeployConfigHFTimes(t *testing.T) {
+	a := &genesis.UpgradeScheduleDeployConfig{
+		// include regolith to demonstrate how unknown destination fields
+		// are not copied
+		L2GenesisRegolithTimeOffset: new(hexutil.Uint64),
```

Comment:
Do we want to silently ignore fields like this? I want to make sure we would catch a situation where the next hardfork is missing from the SCR structs.

Reply:
I can update this so that it panics.

Reply:
it returns an error now